next.js

```diff
@@ -0,0 +1,9 @@
+require('./patch');
+
+/** @type {import('eslint').Linter.Config} */
+module.exports = {
```

Comment:
Should include https://nextjs.org/docs/basic-features/eslint#eslint-plugin ?

Reply:
I believe that is included in `@next/next/core-web-vitals`